jetty-ee10/jetty-ee10-tests/jetty-ee10-test-sessions/jetty-ee10-test-sessions-common/src/test/java/org/eclipse/jetty/ee10/session/AsyncTest.java

```diff
@@ -130,8 +130,11 @@ public void testSessionWithAsyncComplete() throws Exception
             String id = SessionTestSupport.extractSessionId(sessionCookie);
 
             //session should now be evicted from the cache after request exited
-            assertFalse(contextHandler.getSessionHandler().getSessionCache().contains(id));
             assertTrue(contextHandler.getSessionHandler().getSessionCache().getSessionDataStore().exists(id));
+            Awaitility.await().atMost(30, TimeUnit.SECONDS)
+                .until(() -> !contextHandler.getSessionHandler().getSessionCache().contains(id));
+            assertFalse(contextHandler.getSessionHandler().getSessionCache().contains(id));
```

Comment:
just remove the second `assertFalse` as it's already done by Awaitility

Reply:
Done. However, my changes seem to not be showing up on github?! See commit `a2c93bdc70]`